README.md

```diff
@@ -41,33 +41,56 @@ instead of
 
 ## Syntax
 
-_NOTE: this section is a bit in flux after recent changes. Please use the latest released-to-npm release for a stable syntax; we promise we will nail everything down before doing another release._
-
 ### Top-Level Constructs
+The following is a partial EBNF for the top-level constructs in Ecmarkdown.
+
+```
+Document = Paragraph , [ { "\n\n" , Paragraph  } ] ;
 
-An Ecmarkdown **document**, processed using the `ecmarkdown.document(string)` export, is composed of multiple pararaphs.
+Paragraph = OrderedList | UnorderedList | NonList ;
 
-A **paragraph** can be either a list or non-list paragraph.
+OrderedList = OrderedListItem , [ { "\n" , OrderedListItem | UnorderedListItem } ] ;
 
-A **list** is composed of one or more **list items**, which are segments. Non-list paragraphs are also segments.
+UnorderedList = UnorderedListItem , [ { "\n" , OrderedListItem | UnorderedListItem } ] ;
 
-A **segment** is a list of non-formatting characters (i.e., literal text) and formatting characters.
+OrderedListItem = { white space } , { digit } , "." , space , Fragment ;
 
-An Ecmarkdown **fragment**, processed using the `ecmarkdown.fragment(string)` export, is a segment that is assumed to contain no HTML. Thus, you can pass in `` "`2 < 3`" `` as a fragment, and you'll get back as the output HTML `"<code>2 &lt; 3</code>"`.
+UnorderedListItem = { white space } , "*" , space , Fragment ;
 
-Ecmarkdown also allows HTML comments anywhere that does not otherwise disturb the parsing. They are included in the resulting output, in the equivalent location.
+NonList = Fragment ;
+
+Fragment = { StarFormat | UnderscoreFormat | TildeFormat
+         | TickFormat | PipeFormat | Text } ;
+         // could consider renaming to output element
+
+StarFormat = "*" , Text , "*" ;
+// With some complex restrictions on surrounding context to match gmd semantics
```

Comment:
GitHub actually refers to it as "GitHub Flavored Markdown," or GFM. See https://help.github.com/articles/github-flavored-markdown/